README.md

```diff
@@ -1,2 +1,25 @@
-# HW01
-Edit the readme
+---
+title: HW01 
+author: Bei Wang
+date: "July 4, 2020"
+
+---
+
+## About Myself
+I am an incoming PhD student of Population Health Sciences in the track of epidemiology. I grew up in [*Chengdu*](https://en.wikipedia.org/wiki/Chengdu) and lived there for more than 20 years. After earning a master's degree in public health from tht University of Michigan (Ann Arbor) in 2016, I worked closely with the Chinese older immigrants in Chicago as a research assistant. My long-term research interest and goal is to enhance the health and wellbeing of diverse populaitons by understanding and intervening on the predictable and modifiable aspects of health risks in the context of aging and immigration. We live at south loop of Chicago with a basset hound girl whose name is **Margie**. I'm obssessed with the summer lake front trail of Chicago. 
+
+Anyways, something I'm really excited about research- or school-wise are: 
+
+* Age-related health outcomes such as cognitive function decline 
+
+* Prognosis of health conditions such as cancer  
+
+* Immigration and health 
+
+* Longitudinal data 
+
+I figured I should add a picture of **Margie** as this is kinda important. 
```

Comment:
So cute!

Reply:
Well, it looks like the code in my last comment doesn't work either... I borrowed the following code from Summer's README, I think it should work this time.
<img src="https://github.com/beiwo07/HW01/blob/master/image0.png">

Reply:
No, it doesn't work...

Reply:
I also cannot see the picture...But I cannot see Mingyu's picture either, so it might be my problem.